CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ This CHANGELOG follows the format listed [here ](https://github.com/sensu-plugin
 
 ## [Unreleased]
 
+### Changed
```

Comment:
Normally this would be `### Added` as its a new feature but by adding a default this makes it a `### Breaking Changes` as this affects how current users are expecting it to work.